cromulent/vocab.py

```diff
@@ -643,18 +645,24 @@ def add_linked_art_boundary_check():
 
 	def my_linked_art_boundary_check(self, top, rel, value):
 		# True = Embed ; False = Split
-		if isinstance(value, LinguisticObject) and hasattr(value, 'classified_as') and instances['brief text'] in value.classified_as:
-			# linguistic objects and * can be described by embedded linguistic objects
-			return True
+		if isinstance(value, LinguisticObject) and hasattr(value, 'classified_as'):
+			for ca in value.classified_as:
+				if hasattr(ca, 'classified_as') and instances['brief text'] in ca.classified_as:
```

Comment:
This is a stylistic choice, but this 2-part conditional could be simplified to a single `in` by using:

```
instances['brief text'] in getattr(ca, 'classified_as', [])
```